pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
(well, ok, that's still pretty fragile... I'd initially written "if it's silent or prints something to stderr", but then I realized maybe something else could go wrong so that's not a good check either.)

Maybe the fix is to quickly get them to add a --version flag, and then if `iptables-restore --version` prints *anything*, you know it's good...